meson.build

```diff
@@ -1,4 +1,4 @@
-project('libfuse3', 'c', version: '3.3.0',
+project('libfuse3', 'c', version: '3.4.0',
```

Comment:
Could you please also bump `FUSE_USE_VERSION` throughout the codebase, and merge the API bump commit with the commit adding the feature ("libfuse: add copy_file_range() support")?

Reply:
Rebased and squashed the version change.

Did you mean the `FUSE_USE_VERSION` in the `meson.build` files, only? I did not notice others that need updating.

Reply:
The others don't stricly *need* updating, but I would like to keep them at the most recent version so that it doesn't appear that the examples are relying on behavior that has since changed:

```
$ ack FUSE_USE_VERSION
test/test_write_cache.c
10:#define FUSE_USE_VERSION 30

test/test_setattr.c
10:#define FUSE_USE_VERSION 30

include/fuse_common.h
759:#if !defined(FUSE_USE_VERSION) || FUSE_USE_VERSION < 30

include/fuse_lowlevel.h
16: * IMPORTANT: you should define FUSE_USE_VERSION before including this
21:#ifndef FUSE_USE_VERSION
22:#error FUSE_USE_VERSION not defined
1884:#if FUSE_USE_VERSION < 32

include/fuse.h
16: * IMPORTANT: you should define FUSE_USE_VERSION before including this header.
885:#if FUSE_USE_VERSION == 30
981:#if FUSE_USE_VERSION < 32

include/cuse_lowlevel.h
15:#ifndef FUSE_USE_VERSION
16:#define FUSE_USE_VERSION 29

example/hello.c
22:#define FUSE_USE_VERSION 31

[...]
```

Reply:
And now with updated `examples/` :-)